closed/openj9_version_info.h.in

```diff
@@ -31,5 +31,6 @@
 #define OPENJDK_SHA               "@OPENJDK_SHA@"
 #define OPENJDK_TAG               "@OPENJDK_TAG@"
 #define J9JVM_VERSION_STRING      "@OPENJ9_VERSION_STRING@"
+#define J9JDK_EXT_VERSION         "@J9JDK_EXT_VERSION@"
```

Comment:
Sorry, my earlier comment was suggesting to change the name of the define, not the internal variable. The internal variable is not related to J9. The define is for J9 given this file is *openj9*_version_info.h

Reply:
Hmm nm, I guess the internal variable is for J9 as well, give the official name of the extensions.